tests/test_users/test_schemas.py

```diff
@@ -1,28 +1,23 @@
 import copy
 from typing import Any
-from datetime import date
+from datetime import datetime, timedelta
 
 import pytest
 from pydantic.error_wrappers import ValidationError
 
 from src.apps.user.schemas import UserRegisterSchema
+from src.core.factories import UserFactory
 
-
-
-def test_passwords_are_not_identical(
-    register_data: dict[str, Any]
-):
-    user_data = copy.copy(register_data)
-    user_data['password_repeat'] = user_data['password'] + "abcde"
-    with pytest.raises(ValidationError) as exc:
+@pytest.mark.parametrize('incorrect_password', 'passwordX')
+def test_passwords_are_not_identical(incorrect_password):
+    with pytest.raises(ValidationError):
+        user_data = UserFactory.build(password="password", password_repeat=incorrect_password)
         UserRegisterSchema(**user_data)
-
-def test_date_is_from_future(
-    register_data: dict[str, Any]
-):
-    user_data = copy.copy(register_data)
-    user_data['birth_date'] = '9922-02-12'
-    with pytest.raises(ValidationError) as exc:
+        
+@pytest.mark.parametrize('future_date', [datetime.now() + timedelta(days=1)])
```

Comment:
To samo co wyzej, parametryzacja ma sens tylko jak jest wiecej niz jedna wartosc jako parametr